test/tools/Modules/HelpersCommon/HelpersCommon.psm1

```diff
@@ -365,19 +365,29 @@ function New-ComplexPassword
 }
 
 # return a specific string with regard to platform information
-function Get-PlatformInfo
-{
-    if ( $IsWindows ) {
-        return "windows"
+function Get-PlatformInfo {
+    $retObj = if ( $IsWindows ) {
+        return @{Platform = "windows"; Version = '' }
```

Comment:
no need to assign to `$retObj` if you return from this `if` block.

Reply:
Fixed.